src/app/pages/Designathons/Designathon24/components/About/index.jsx

```diff
@@ -0,0 +1,58 @@
+import cn from "./About.module.scss";
+import clsx from "clsx";
+
+import airplane from "../../assets/graphics/about/airplane.svg";
+import paperclip from "../../assets/graphics/about/paperclip.svg";
+import tr_gradient from "../../assets/graphics/about/tr_gradient.svg";
+import bl_gradient from "../../assets/graphics/about/bl_gradient.svg";
+import tr_stars from "../../assets/graphics/about/tr_stars.svg";
+import bl_stars from "../../assets/graphics/about/bl_stars.svg";
+import Notecard from "../Notecard/Notecard";
+
+const About = () => {
+	return (
+		<div className={cn.container} id="s-about">
+			<h2 className={cn.aboutHeading}>About</h2>
+			<Notecard>
+				<p>
+					Design-a-thon is a 3 day long hybrid event where you or a team design
+					a product focused on the theme: community and inclusivity. It takes
+					place on February 24, starting at 5:30 pm and ends February 26, 8:00
+					pm PST.
```

Comment:
Note: presuming language will still need to be updated with the correct dates?

Reply:
Oh yes, we should circle back to this as I don't believe all the copywriting has been completed yet.

Here's the document that was shared with me for text content: https://docs.google.com/document/d/1jeIarZvbfW_U1wC4599xUgyQoYoy17G3fcMAXM8xGaI/edit

It hasn't been fully updated yet